Gemfile

```diff
@@ -1,8 +1,8 @@
-source "http://mirror.ops.rhcloud.com/mirror/ruby/"
+source "https://rubygems.org"
 ruby "2.2.3" # this should match `.ruby-version` and docs/setup.md
 
 gem "active_model_serializers"
-gem "activeadmin", github: "activeadmin"
+gem "activeadmin", github: "activeadmin", ref: "d787029e"
```

Comment:
Wasn't sure if there was a reason for using the latest from GitHub, but since the latest from `master` isn't necessarily stable, I was nervous about upgrading it. We should probably add a comment here about why we are using this instead of a released gem, but wasn't sure for the reason.